implementations/micrometer-registry-kairos/src/main/java/io/micrometer/kairos/KairosConfig.java

```diff
@@ -0,0 +1,56 @@
+package io.micrometer.kairos;
```

Comment:
could you add the Pivotal license to all source files?

Reply:
done